composer.json

```diff
@@ -9,15 +9,15 @@
         "php": ">=7.1",
         "ext-rdkafka": "*",
         "ext-json": "*",
-        "illuminate/support": "~5.5.0||~5.6.0||~5.7.0",
-        "illuminate/console": "~5.5.0||~5.6.0||~5.7.0",
-        "illuminate/config": "~5.5.0||~5.6.0||~5.7.0"
+        "illuminate/support": "~5.5.0||~5.6.0||~5.7.0||~5.8.0",
```

Comment:
Does it still work with 5.5? If it do, it should be together with #53 
Otherwise, remove the incompatible versions.

Reply:
This PR is pointing to #53